src/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.cs

```diff
@@ -86,32 +86,33 @@ internal class SafeGssCredHandle : SafeHandle
         /// </summary>
         public static SafeGssCredHandle Create(string username, string password, string domain)
         {
-            SafeGssCredHandle retHandle = null;
```

Comment:
Just FYI, you can tag "w=1" on to the query string to see a diff with whitespace-differences ignored, e.g. https://github.com/dotnet/corefx/pull/6111/files?w=1